v2/api/appconfiguration/v1api20220501/structure.txt

```diff
@@ -17,12 +17,14 @@ github.com/Azure/azure-service-operator/v2/api/appconfiguration/v1api20220501
 │   │   │   └── KeyVaultProperties: *Object (2 properties)
 │   │   │       ├── IdentityClientId: *string
 │   │   │       └── KeyIdentifier: *string
-│   │   ├── Identity: *Object (1 property)
-│   │   │   └── Type: *Enum (4 values)
-│   │   │       ├── "None"
-│   │   │       ├── "SystemAssigned"
-│   │   │       ├── "SystemAssigned, UserAssigned"
-│   │   │       └── "UserAssigned"
+│   │   ├── Identity: *Object (2 properties)
```

Comment:
I've gone through and confirmed that there are no breaking changes (only addition of a new property with this shape) for all types, but good to get a second set of eyes on it to confirm.

The properties should all look like this too:
```
│   │   │   └── UserAssignedIdentities: Object (1 property)[]
│   │   │       └── Reference: genruntime.ResourceReference
```